llvm/lib/Target/RISCV/GISel/RISCVLegalizerInfo.cpp

```diff
@@ -582,12 +582,19 @@ RISCVLegalizerInfo::RISCVLegalizerInfo(const RISCVSubtarget &ST)
       .libcallFor(ST.is64Bit(), {{s32, s128}, {s64, s128}});
 
   // FIXME: We can do custom inline expansion like SelectionDAG.
-  // FIXME: Legal with Zfa.
   getActionDefinitionsBuilder({G_FCEIL, G_FFLOOR, G_FRINT, G_FNEARBYINT,
                                G_INTRINSIC_TRUNC, G_INTRINSIC_ROUND,
                                G_INTRINSIC_ROUNDEVEN})
+      .legalFor(ST.hasStdExtZfa() /*ST.hasStdExtF()*/, {s32})
```

Comment:
Is it intentional for ST.hasStdExtF to be commented out? I am a little confused on its meaning as a comment.

Reply:
I'd probably drop it, or change it to `/*hasStdExtF() is implied by hasStdExtZfa*/`

Reply:
I'm going to drop it, comments arent checked by the compiler its probably just more confusing to include it